js/ui/scroll_view/ui.scrollable.js

```diff
@@ -270,7 +270,7 @@ var Scrollable = DOMComponent.inherit({
 
     _attachNativeScrollbarsCustomizationCss: function() {
         // NOTE: Customize native scrollbars for dashboard team
-        if(!(navigator.platform.indexOf('Mac') > -1 && browser['webkit'])) {
+        if(devices.real().deviseType === "desctop" && !(navigator.platform.indexOf('Mac') > -1 && browser['webkit'])) {
```

Comment:
This change should be tested. 
deviseType -> deviceType
"desctop" -> "desktop"